packages/@ama-sdk/client-angular/project.json

```
@@ -45,5 +45,8 @@
     "prepare-build-builders": {},
     "build-builders": {}
   },
-  "tags": []
+  "tags": [
+    "scope:ama-sdk",
+    "type:sdl-client"
```

```suggestion
    "type:sdk-client"
```
We may want a linting process to avoid these typos